src/components/ConstitutionalCommittees/Overrall/styles.tsx

```diff
@@ -2,10 +2,10 @@ import { Grid, styled } from "@mui/material";
 interface CardItemProps {
   length: number;
   wide?: number;
-  itemOnRow: number;
+  itemonrow: number;
```

Comment:
For naming variables please use camelCase.

Reply:
https://cardanofoundation.atlassian.net/browse/MET-2303. in ticket 2303 we will fix and remove all console error and warning and styled component in MUI ui accept props with lowercase and use camelCase in console will waring